assets/coffee/modules/EDWARE.individualStudent.coffee

```diff
@@ -239,7 +246,7 @@ define [
           block_info = {'grade': @configData.labels.grade + " " + asmt_grade,
           'date_taken': edwareUtil.formatDate(assessment['date_taken']),
           'name': assessment['claims'][0]['name'],
-          'desc': assessment['claims'][0]['perf_lvl_name'],
+          'desc': performance_level_description_by_level[assessment['claims'][0]['perf_lvl']],
```

Comment:
Agreed. Lookup by performance level is preferable, but for consistency I will make the suggested change.